src/main/java/com/yandex/money/api/methods/AccountInfo.java

```diff
@@ -99,6 +99,12 @@
      */
     public final List<YandexMoneyCard> yandexMoneyCards;
 
+    /**
+     * list of Yandex.Money virtual cards
+     */
+    public final List<YandexMoneyCard> virtualCards;
```

Comment:
This is not a list of YandexMoneyCard entities. This is a completely new entity, that must be created.